packs/feats/united-assault.json

```diff
@@ -24,7 +24,41 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Knights of Lastwall"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
```

Comment:
```suggestion
                "domain": "attack-roll",
```

Reply:
Wrong selector anyways, I was doing it while GMing